homeassistant/components/switchbee/__init__.py

```diff
@@ -0,0 +1,137 @@
+"""The SwitchBee Smart Home integration."""
+
+from __future__ import annotations
+
+import logging
+
+import switchbee
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, SCAN_INTERVAL
```

Comment:
```suggestion
from .const import DOMAIN, SCAN_INTERVAL_SEC
```
also make sure to import `timedelta`